focus-areas/health-101/README.md

```diff
@@ -0,0 +1,19 @@
+# Sustainability
+
+Goal: We have a set of light-weight metrics and methods required to evaluate the very basic health of an open source project.
+
+# Some Potential User Stories
+
+
+* As a potential consumer of an open source project I want to evaluate the health of engagement around that project before ingesting/using.
```

Comment:
should we give them a explicit roles, like OSPOs, Corporations, individual users etc...